contracts/BadgerVotingShare.sol

```diff
@@ -323,6 +343,7 @@ contract BadgerVotingShare {
             _sushiswapBalanceOf(_voter) +
             _rariBalanceOf(_voter) +
```

Comment:
I don't like that we have rari here, I haven't checked but since rari got hacked, exchange rates may be wrong
Also if Rari is upgradeable, this could eventually cause a vulnerability to us due to private key leak

Can we remove rari safely?

Reply:
4861e8e